main.go

```diff
@@ -655,13 +655,17 @@ func insertNewItems(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	password := flag.String("password", "", "password for mysql db")
+	env := flag.String("env", "development", "Environment [development|staging|production]")
```

Comment:
I was actually toying with the idea of enforcing password supply via an ENV variable:

```
DB_PASS=supersecret ./point-of-sale
```

That would do away with the concern about password storage in the app itself, as well as allowing for scripted application starts.

As for allowing an `env` flag to toggle a few settings at a time, I do think that the ability to do so is a good idea, but as elsewhere, this is your baby.
I'll rip it out if you think it's just adding confusion.